source/manage-apps/deploy/deployment-models-and-regions.txt

```diff
@@ -32,65 +32,175 @@ Deployment Models
 Global Deployment
 ~~~~~~~~~~~~~~~~~
 
-Globally deployed App Services Apps are hosted in every :ref:`cloud
-region <realm-regions>` that App Services supports. The a Realm SDKs
-automatically execute application requests, such as reading user
-information or calling a function in the cloud region that's closest to
-the user that issued the request.
-
-Write operations to a linked MongoDB cluster are handled within a single
-region that you specify when you :ref:`create your application
-<create-a-realm-app>`. By default, App Services handles writes in
-the region closest to your application's original linked MongoDB
-cluster.
+A globally deployed app is available in multiple regions across the
+world. Any global region can receive and process incoming requests for a
+global app.
+
+All write operations to a linked data source are handled within a single
```

Comment:
i think this line could be clarified. this is just for write operations for a linked **MongoDB** data source, i assume?